s3torchconnector/tst/unit/test_s3_client.py

```diff
@@ -48,3 +50,18 @@ def test_list_objects_log(s3_client: S3Client, caplog):
     with caplog.at_level(logging.DEBUG):
         s3_client.list_objects(TEST_BUCKET, TEST_KEY)
     assert f"ListObjects {S3_URI}" in caplog.messages
+
+
+def test_s3_client_default_user_agent():
+    s3_client = S3Client(region=TEST_REGION)
```

Comment:
There's no test validating the user agent is correctly passed to out Mountpoint client